drivers/dac/ltc2672/ltc2672.c

```diff
@@ -139,39 +139,125 @@ int ltc2672_transaction(struct ltc2672_dev *device, uint32_t comm, bool is_32)
 }
 
 /**
- * @brief set the current for a selected DAC channel
+ * @brief get dac code value from current of a selected DAC channel
  * @param device - ltc2672 descriptor
- * @param current - current to set in mA
- * @param out_ch - channel to set the current
+ * @param dac_current - current in mA
+ * @param out_ch - DAC channel
+ * @return code value
+ */
+uint32_t ltc2672_current_to_code(struct ltc2672_dev *device, float dac_current,
+				 enum ltc2672_dac_ch out_ch)
+{
+	float current;
+	uint32_t current_code;
+
+	current = dac_current / device->max_currents[out_ch];
+
+	if (device->id == LTC2672_12)
+		current_code = (uint32_t)(current * (float)(LTC2672_12BIT_RESO));
+	else
+		current_code = (uint32_t)(current * (float)(LTC2672_16BIT_RESO));
+
+	return current_code;
+}
+
+/**
+ * @brief get current from dac code of a selected DAC channel
+ * @param device - ltc2672 descriptor
+ * @param dac_code - dac code
+ * @param out_ch - DAC channel
+ * @return current value in mA
+ */
+float ltc2672_code_to_current(struct ltc2672_dev *device, uint16_t dac_code,
```

Comment:
ditto: avoid float type

Reply:
Removed this function as this is not required and conversions can be done easily without this function.